airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/tickets.json

```diff
@@ -179,6 +179,26 @@
     },
     "forum_topic_id": {
       "type": ["null", "integer"]
+    },
+    "custom_status_id": {
+      "type": ["null", "integer"]
+    },
+    "fields": {
```

Comment:
Where did you saw the `fields` value in Zendesk API documentation? This shouldn't be `custom_fields`?

Reply:
Last month the zendesk team made a change to the tickets that are generated by the chat system.
Today there are new fields that are being made available by the API at the /api/v2/tickets.json endpoint so that we can identify these tickets (which are generated via the chat system), the new field is from_messaging_channel

![Screenshot 2023-04-14 at 17 08 44](https://user-images.githubusercontent.com/40533438/232145947-dbc6f08c-97f4-4b0c-84b2-9c8b78c0d186.png)

Reply:
You can get this information in link https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#list-tickets

Reply:
I can see the field `from_messaging_channel` in Zendesk API documentation but there isnt any `fields` there.

Reply:
there where?

Reply:
In the response for `ticket` endpoint.

Reply:
I made a test request and this field appeared.


<img width="395" alt="Screenshot 2023-04-14 at 17 18 51" src="https://user-images.githubusercontent.com/40533438/232147041-4a3875c2-d64f-40e2-98dc-396ae1403957.png">

Reply:
@asafepy the problem is not the field `from_messaging_channel` you have added a new field/column called `fields`...
You said in the description
> The fields, custom_status_id, fields, from_messaging_channel were added and I made an add in the Ticket Stream schema to map this new data.

What I'm saying is for the **fields** I don't see this key in tickets object response.

Reply:
aaah, ok... I'm understanding...
but this field is coming in the answer... in mine... and I checked the documentation and it is not described at all. very strange... I'm going to remove it from the schema.


<img width="455" alt="Screenshot 2023-04-14 at 17 27 12" src="https://user-images.githubusercontent.com/40533438/232148351-7fb00118-84e2-407e-9fe7-8bfc4c5c2244.png">